lib/monitoring/ci-alarms.ts

```diff
@@ -39,7 +39,7 @@ export class JenkinsMonitoring {
 
     this.alarms.push(new Alarm(stack, 'MainNodeHighCpuUtilization', {
       alarmDescription: 'The jenkins process is using much more CPU that expected, it should be investigated for a stuck process/job',
-      metric: mainNode.ec2InstanceMetrics.cpuTime.with({ statistic: 'max' }),
+      metric: mainNode.ec2InstanceMetrics.cpuTime.with({ statistic: 'avg' }),
```

Comment:
Hey @rishabh6788 can you please post a alarm details from your testing which gets created with `avg` CPUUtilization.
Thank you

Reply:
<img width="858" alt="Screenshot 2022-07-27 at 7 31 50 PM" src="https://user-images.githubusercontent.com/19776689/181266655-7309a4ba-78df-46f9-a92a-5a2777798907.png">
Attached screenshot of CPU Usage alarm that got created.